providers/fab/src/airflow/providers/fab/www/templates/airflow/token_set.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Redirecting...</title>
+  <script type="text/javascript">
+    document.addEventListener("DOMContentLoaded", function () {
+      localStorage.setItem("token", JSON.stringify("{{ token }}"));
+      window.location.href = "{{ redirect_url }}";
+    });
```

Comment:
Although the page is small, there's no really point waiting for DOMContentLoaded is there? Just do this all directly here in the script tag?